### ISSUE TITLE
Form should remember place if user clicks away from it (e.g., Learn more under processing methods)

### ISSUE THREAD

**Issue by dbranchini** [sul-dlss/sul-requests#2419]
### Steps to reproduce:

1. I select pickup physical item, click continue, get the option to expedite/wait 
2. I clicked "Learn more" 
3. Then used the back button to get back to the request page

### Actual result:

It goes back, but now it's back on step one with "pickup physical item" still selected and the continue button is disabled. I can get around it by selecting email and then pickup again. 

### Expected result:

It shouldn't lose the user's place. I'd expect it to still be on step 2 with the expedite/wait options since that's where I left it when I clicked "Learn more."

### Record and other information:

https://requests.stanford.edu/patron_requests/new?instance_hrid=1849058&origin_location_code=SAL3-STACKS

![Image](https://github.com/sul-dlss/sul-requests/assets/6816624/e1840004-6cda-40cd-b9a9-77be1aed9e5b)

**Comment by cbeer** [comment]
I seem to remember different browsers doing different things, and having to do something to clean up the form if the user refreshed the page.

Can these links for more information just open in a new tab/window?

**Comment by dbranchini** [comment]
Yes, I know a lot of people frown upon that, but especially in this case when you're filling out a form. I'd prefer to not lose your place in the form.

**Comment by dbranchini** [comment]
Funny that Niqui couldn't recreate because I was using Chrome.
